content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -36,12 +36,15 @@ <h3>Scaling an application</h3>
               {{< /note >}}
             </div>
             <div class="col-md-4">
-                <div class="content__box content__box_lined">
-                    <h3>Summary:</h3>
+                <div class="content__box_lined">
```

Comment:
The changes don't appear suitable to me. They've removed the HTML class `content__box ` (_used for margin and padding)_ and replaced it with manual spacing using `<br>` and `&nbsp` . This approach, while functional, diminishes code readability and maintainability. Consider using CSS based solution for layout and spacing. Manual spacing with non-breaking spaces will also affect screen readers and keyboard navigation, diminishing accessibility.

Reply:
I can see that the manual spacing is gone, but I agree that removing the class isn't the right fix here.

We should check that any fix works as expected across localizations.

Reply:
As I had seen in every localization same issue persist and I had used many most probable solution but this solution work fine

Reply:
if we remove content-box then the padding and margin  will be removed in this div only which i dont think will harm the reader as it will look good then before  as the word breakdown to next line will not be seen further. 

If you have any suggestion then we should go for it.

Reply:
I don't have a specific answer about what CSS / SCSS change is appropriate, but changing the HTML for just one tutorial out of the set is the wrong fix I'm afraid.

You may need to do more work in order to work out a fix that reviewers will accept.

Reply:
Sure I will definately look into it.

Reply:
@dipesh-rawat @sftim  I find the solution here i need to add custom css and i am planning to use it into the same file under <style></style> as the page is designed in bootstrap and no relevant css and scss file is present for it. 
If it look goods then i can go ahead.